src/io/sarnowski/swagger1st/schemas/swagger_2_0.clj

```diff
@@ -1,5 +1,6 @@
 (ns io.sarnowski.swagger1st.schemas.swagger-2-0
-  (:require [schema.core :as s]))
+  (:require [clojure.string]
```

Comment:
I've seen `:require [clojure.string :as string]` in this project more often, would be cool to have it consistent and I am also fan of more explicit things.